laserstudio/__main__.py

```diff
@@ -68,4 +69,8 @@
     win = LaserStudio(yaml_config)
     win.setWindowTitle(app.applicationDisplayName())
     win.show()
-    sys.exit(app.exec())
+    return app.exec()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Related to previous comment:
`sys.exit(main())` ?

Reply:
Yes.